presidio-analyzer/analyzer/context_simplifier/nlp_context_simplifier.py

```diff
@@ -0,0 +1,39 @@
+import os
+import logging
+import spacy
+from analyzer.context_simplifier.context_simplifier import ContextSimplifier
```

Comment:
better to use __init__ to have more clear imports 
```suggestion
from analyzer.context_simplifier import ContextSimplifier
```